### ISSUE TITLE
Possible bug when widget updates

### ISSUE THREAD

**Issue by alekseifm** [pedrohff/flutter_heatmap_calendar#7]
Awesome package! Thanks!

I am using the HeatMapCalendar, but sometimes when it updates, values disappear and dates are shifted by one.
Can someone help me or it is a known bug?

Thanks!

![image](https://user-images.githubusercontent.com/68206072/92987531-a5611c80-f499-11ea-8d62-bd2b47f30388.png)
![image](https://user-images.githubusercontent.com/68206072/92987538-af831b00-f499-11ea-9d03-6f090c7060a5.png)

**Comment by draftin1** [comment]
Yeah, the state doesn't load the data properly. Sometimes it's there and sometimes it doesn't.

**Comment by cd104** [comment]
Appears to be caused by a typo in TimeUtils... 

```
/// Returns date without timezone info (UTC format)
  static DateTime removeTZ(DateTime dateTime) {
    return DateTime.utc(dateTime.year, dateTime.month, dateTime.day, dateTime.hour, dateTime.minute,
        dateTime.second, dateTime.millisecond, dateTime.millisecond); <<<< should be dateTime.microsecond
  }
```

**Comment by draftin1** [comment]
That's it!
